tools/eubfr-cli/commands/delete.js

```diff
@@ -0,0 +1,114 @@
+const url = require('url');
+const path = require('path');
+
+const aws4 = require('aws4');
+const dotenv = require('dotenv');
+const elasticsearch = require('elasticsearch');
+const request = require('request-promise-native');
+
+const getServiceLocation = require('../lib/getServiceLocation');
+
+dotenv.config({
```

Comment:
`dotenv` should only be called in the main `cli.js` file. Then, you could write the commands as pure functions 😉